src/app/modules/highlight/top-highlight-cards/top-highlight-cards.component.scss

```diff
@@ -64,10 +86,13 @@
       display: flex;
       margin-top: 20px;
       max-width: 100%;
-      -webkit-line-clamp: 3;
       flex-direction: column;
       overflow: hidden;
-      text-overflow: ellipsis;
+      hyphens: auto;
+
+      @include media-breakpoint-down(sm){
+        max-height: 110px;
+      }
```

Comment:
This produces results like this: 

![image](https://user-images.githubusercontent.com/8917249/94583597-ee78e500-027d-11eb-8e02-db7c46c4f9f4.png)

I think you're trying to solve the wrong problem here. This doesn't look quite right to me. Try to use the project's `shortDescription` or figure out how exactly you want to showcase highlighted projects, instead of continuing to work with content that's not fit for this case 😄 .

Reply:
I don't agree you shouldn't try to hide everything on the page on mobile that means that the info isn't relevant enough to show on the big screens ass well. and the description length doesn't bother me

Reply:
When you remove more of the description I think the card isn't useful anymore, it will barely display any info about the project.

Reply:
In the next release, there will be something called highlight descriptions(#245). This should help with the content

Reply:
@MaxKorlaar, thnx for your feedback. Could you take another look at my new pr? I made sure the text wouldn't be cut off by showing all text. This gave a problem that not all cards had the same height. So I choose right now that all cards should have the same height as the largest card. So on the larger viewports, the cards have the same height (on smaller they can have different heights). I also updated it to the sort description. Do you think this is an improvement, or would you say the older version would work better? Thnx in advance again 😉